modules/dex/openid.go

```diff
@@ -0,0 +1,19 @@
+package dex
+
+type OpenIDConfiguration struct {
```

Comment:
suggestion: add documentation comment, required for all public methods, variables, constants and types.